src/components/withWindowDimensions.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable react/no-unused-state */
-import React, {forwardRef, createContext} from 'react';
+import React, {forwardRef, createContext, useMemo} from 'react';
 import PropTypes from 'prop-types';
-import {Dimensions} from 'react-native';
+import {useSafeAreaFrame} from 'react-native-safe-area-context';
```

Comment:
pardon me for my ignorance, this change will impact all the screens, so are we sure this works well on RNW?

Reply:
It has a web implementation, I did test on web and it seems to work well. https://github.com/th3rdwave/react-native-safe-area-context/blob/main/src/NativeSafeAreaProvider.web.tsx.